bentoml/_internal/io_descriptors/numpy.py

```diff
@@ -199,18 +159,25 @@ def __init__(
 
         self._sample_input = None
 
-        # whether to use packed representation of numpy while sending protobuf
-        # this means users should be using raw_value instead of array_value or multi_dimensional_array_value
-        self._packed = packed
         if bytesorder and bytesorder not in ["C", "F", "A"]:
             raise BadInput(
                 f"'bytesorder' must be one of ['C', 'F', 'A'], got {bytesorder} instead."
             )
         if not bytesorder:
-            bytesorder = "C"  # default from numpy (C-order)
             # https://numpy.org/doc/stable/user/basics.byteswapping.html#introduction-to-byte-ordering-and-ndarrays
+            bytesorder = "C"  # default from numpy (C-order)
+
         self._bytesorder: t.Literal["C", "F", "A"] = bytesorder
```

Comment:
remove _bytesorder?

Reply:
_bytesorder can be used for np.frombuffer.